CHANGELOG.md

```diff
@@ -3,7 +3,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
 
-### 🔄 Changed
+## StreamChat
+### ✅ Added
+- Add `completion` to disconnect APIs [#2016](https://github.com/GetStream/stream-chat-swift/issues/2016)
+### ✅ Fixed
+- Fix not waiting for last batch of events to be processed when connecting as another user [#2016](https://github.com/GetStream/stream-chat-swift/issues/2016)
```

Comment:
Did we fix it out-of-the-box, or does the customer need to do anything about it? @evsaev